app/main.py

```diff
@@ -1,13 +1,45 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
```

Comment:
Look , you have 2 places where this code can be reused . Always try to stick DRY ( Do not repeat yourself ) principle
Just reused ligic of format_linter_error function insted of copying same code : )